goodtablesio/models/job.py

```diff
@@ -110,15 +113,53 @@ def get_ids():
     return [j.id for j in job_ids]
 
 
-def get_all():
-    """Get all jobs in the database as dict.
+def find(filters=None, limit=10, offset=0):
```

Comment:
You are right, in this particular case the interface is pretty terrible and leaks the SA implementation, it was difficult to abstract to native types so perhaps we should move it to the GH models. But these other are superior to vanilla SA on views, tasks, tests etc IMO:

```python

models.jobs.get(job_id)
models.jobs.update({'id': job_id, 'status': 'success'})
models.jobs.get_by_plugin('github', limit=5)

```

I think it's a matter of common sense. Let's stay with these for now and once we have more plugins in place we can decide.

Again we can return SA models from these helpers instead of dicts if you want.